### DISCUSSION TITLE
Release v0.18

### DISCUSSION THREAD

**Discussion by mozman** [mozman/ezdxf#712] [Announcements]
Some highlights:
- support for creating angular-, arc- and ordinate dimensions
- create `MLEADER` entities
- extended entity queries
- add-on `meshex` to  exchange meshes with other tools as `STL`, `OFF` or `OBJ` files
- add-on `openscad`, interface to [OpenSCAD](https://openscad.org/)

For more information see the [release notes](https://ezdxf.mozman.at/release-v0-18.html).

See also #704 - Road to v1.0

**Comment by chibai** [comment]
👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍

**Comment by chibai** [comment]
EZDXF: A Python package for creating and modifying DXF drawings, regardless of the DXF version.
I'm confused how clustering algorithm works on this??? Clustering algorithm is machine learning algorithm

**Comment by mozman** [comment]
You can also use this functions to find groups of points that belong together.

**Reply by chibai** [reply]
well.  make sense 👍
